packages/editor/src/components/rich-text/list-edit.js

```diff
@@ -70,52 +70,54 @@ export const ListEdit = ( { editor, onTagNameChange, tagName, onSyncDOM } ) => (
 			} }
 		/>
 		<BlockFormatControls>
-			<Toolbar
-				controls={ [
-					{
-						icon: 'editor-ul',
-						title: __( 'Convert to unordered list' ),
-						isActive: isActiveListType( editor, 'ul', tagName ),
-						onClick() {
-							if ( isListRootSelected( editor ) ) {
-								onTagNameChange( 'ul' );
-							} else {
-								editor.execCommand( 'InsertUnorderedList' );
-								onSyncDOM();
-							}
+			<div className="list-format-toolbar">
```

Comment:
This does not adhere to CSS naming guidelines, where the class name should follow the pattern:

`[ package name ]-[ directory name ]__[ descendant description ]`

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#naming

Here, I might think something like `editor-rich-text__list-format-toolbar` could be appropriate.

Reply:
I was mimicking this though:

https://github.com/WordPress/gutenberg/blob/652a662a8aba2b621978871e56706990d5deb653/packages/editor/src/components/rich-text/format-toolbar/index.js#L9

Reply:
> I was mimicking this though:

This format toolbar follows the guideline from my previous comment, since it's in the `editor` top level directory, in the `format-toolbar` immediate directory, and is not a descendant but rather the top-level element returned by the component. The last of these wasn't entirely clear in my previous comment, but is explained in the related guideline line. It allows the descendant description to be omitted, thus leaving only `editor-format-toolbar`.